### ISSUE TITLE
Make sure the exported PDF reports complies with PDF-A (Archivable)

### ISSUE THREAD

**Issue by Sebastiaan127001** [ICTU/quality-time#2927]
PDF-A is the standard in NL and Dutch Government, would be great if QT supports this

**Comment by fniessink** [comment]
This should be fixed in Quality-time v5.8.0 with puppeteer 22.0.0 (that contains https://github.com/puppeteer/puppeteer/pull/11778).

**Comment by Sebastiaan127001** [comment]
https://nl.wikipedia.org/wiki/PDF/A

**Comment by Sebastiaan127001** [comment]
The Dutch governmental regulation to which the PDF's must comply: https://www.forumstandaardisatie.nl/open-standaarden/pdf-nen-iso

**Comment by Sebastiaan127001** [comment]
Reports are currently generated in PDF 1.4 and not in PDF/A

![image](https://github.com/ICTU/quality-time/assets/95095453/a0b785fc-5f7a-4922-aed5-f486b0c392a7)

**Comment by fniessink** [comment]
It seems Chromium does not support PDF/A. The weird thing is that the library used by Chromium, Skia, does seem to be able to produce PDF/A (at least this issue is marked as fixed: https://issues.skia.org/issues/40034216), but it's unclear how to force Chromium to let Skia produce PDF/A.

Alternatives considered:
- Convert the PDF produced by Chromium using Ghostscript, but that didn't result in PDF/A either, but PDF 1.7.
- Use LibreOffice, but that seems a rather heavy solution, if it works. Not tested.
- Pdf-lib (Javascript), but it can't generate PDF/A.

**Comment by eric-nieuwland** [comment]
Check out https://github.com/Kozea/WeasyPrint,
documentation at https://doc.courtbouillon.org/weasyprint/stable/index.html

**Comment by fniessink** [comment]
Nice. According to the documentation WeasyPrint should be able to create PDF/A. Unfortunately this requires a rewrite of the renderer component.

Possibly relevant: https://www.digitaleoverheid.nl/nieuws/nieuwe-technieken-beschikbaar-voor-toegankelijke-pdfs/
